### ISSUE TITLE
Balanced Difficulty

### ISSUE THREAD

**Issue by aori6425** [yairm210/Unciv#1884]
I hope this is right place for this question.

I was wondering if it was possible to add an option to create a perfectly balanced difficulty level, meaning AI and human player are playing at the exact same level. From what I understand, PRINCE is kind of that way but the AI still cheats because it always plays CHIEFTAIN and gets happiness, support and other bonuses. (I'm basing this on my understanding of Civ 5 from https://civilization.fandom.com/wiki/Difficulty_level_(Civ5) )

Basically, I think I'm asking for a game setup where both human and AI play on Prince.

Maybe this could be done by adding a checkbox option "No Cheating AI" when setting up the game similar to the mod available for Civ 5.

**Comment by yairm210** [comment]
I'm confused as to what would be the purpose of such a change. What would you gain from this that you don't have in Prince? What gameplay improvement does a 'perfectly balanced' difficulty give? If you want to destroy the AI, then Prince should be almost exactly the same for you

**Comment by aori6425** [comment]
I would like to experience a game where neither me nor the AI gets any "unfair" advantages. I'm usually a defensive player, I focus on research, building and improvement rather than going out and attacking unless I'm provoked first. From what I understand, as the AI starts getting unfair advantages (cheats) this strategy can only fail because the cards are stacked against you long term and I could see that in the game I just played.

I want to see if I can outsmart the AI without having to resort to doing all out war again and againt in order to keep them from getting way ahead from me. I want to see if I can outsmart them in other ways, e.g. when to expand, what policies/research to do at what time, etc. And only the prince mode with no AI cheating will allow me to do that because otherwise, the AI will get ahead no matter what long term.

I don't think that I'm the only one on this, judging from the fact that someone already made a mod for this exact gameplay on the official version. But I do understand that everyone's time is limited and you probably have more pressing things to work on right now :) So you if it's a lot of work you could maybe just keep this for sometime in the future :)

**Comment by Whistler11** [comment]
So like... You want the experience of playing against other human players (PvP), without the other players?

My observation is human minds and an AI are different from the start. Humans just have an advantage, which Prince attempts to compensate for.

**Comment by yairm210** [comment]
After thinking about this for a while, I think I'll close it
Prince is probably as close as you're going to get to balanced, and it's not like the AI is all that great, it's extremely difficult to create a good AI when there are so many aspects of the game to consider
